fendermint/app/settings/src/lib.rs

```diff
@@ -206,6 +206,8 @@ pub struct Settings {
     builtin_actors_bundle: PathBuf,
     /// Custom actors CAR file.
     custom_actors_bundle: PathBuf,
+    /// upgrade_info JSON file.
+    pub upgrade_info: PathBuf,
```

Comment:
```suggestion
    /// upgrade_info JSON file.
    upgrade_info: PathBuf,
```

Don't make it public, because if it's `home_relative!` then people should access it through the `settings.upgrade_info()` method, and that should be the only way, to avoid confusion.